### ISSUE TITLE
Event Log - Include Warning entry type

### ISSUE THREAD

**Issue by Koubek** [Microsoft/nav-docker#151]
I have a question - would you mind consider adding **Warning** messages to the container log output?

I have found that we are missing some relevant information being reported as a Warning message. For example:
```
Index              : 1700
EntryType          : Warning
InstanceId         : 218
Message            : Server instance: NAV
                     Tenant ID: 
                     <ii>User: 
                     Type: Microsoft.Dynamics.Nav.Types.Exceptions.NavAppException
                     SuppressMessage: False
                     DiagnosticsSuppress: True
                     SuppressExceptionCreatedEvent: False
                     FatalityScope: None
                     Message: <ii>The application object of type 'Table' with the ID '50130' is defined in the app 'ALProject111 by Default publisher 1.0.0.0', but already exists in the base application.</ii>
```

I think the following should be OK:
```PowerShell
Get-EventLog -LogName Application -After $lastCheck -ErrorAction Ignore | Where-Object { ($_.Source -like '*Dynamics*' -or $_.Source -eq $SqlServiceName) -and ($_.EntryType -eq "Error" -or $_.EntryType -eq "Warning") -and $_.EntryType -ne "0" } | Select-Object TimeGenerated, EntryType, Message | format-list
```

**Comment by freddydk** [comment]
WIll be included in generic 0.0.4.5

**Comment by freddydk** [comment]
Generic 0.0.4.5 is out there - images being generated for all builds
